docs/UserGuide.md

```diff
@@ -291,17 +291,17 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS housing/HOUSING_TYPE availa
-| `ANIMAL_NAME`    | - If `availability/NotAvailable`: Name of the animal fostered / nil<br/> - If `availability/Available`: nil<br/> - If `availability/nil`: nil                                                                                                                               | `Fluffball`, `nil`                                          |
-| `TYPE_OF_ANIMAL` | - Type of animal which the fosterer is currently fostering, or prefer to foster<br/> - Case-sensitive<br/> - If `availability/NotAvailable`: current.Dog / current.Cat / nil<br/> - If `availability/Available`: able.Dog / able.Cat / nil<br/> - If `availability/nil`: nil | `current.Dog`, `current.Cat`, `able.Dog`, `able.Cat`, `nil` |
-| `TAG`            | Tag to be associated with the fosterer                                                                                                                                                                                                                                      | `experienced`, `urgent`                                     |
+| Parameter        | About                                                                                                                        | Example                                                     |
+|------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `NAME`           | Name of the fosterer                                                                                                         | `Alice Tan`, `Harry Yeo`                                    |
+| `PHONE_NUMBER`   | Phone number of the fosterer                                                                                                 | `93456778`, `89067547`                                      |
+| `EMAIL`          | Email of the fosterer                                                                                                        | `thomas718@gmail.com`, `kate@yahoo.com.sg`                  |
+| `ADDRESS`        | Address of the fosterer                                                                                                      | `Orchard road, Blk 8, #13-04`                               |
+| `HOUSING_TYPE`   | - Housing type of the fosterer<br/> - Case-sensitive<br/> - Can be HDB / Condo / Landed / nil                                | `HDB`, `Condo`, `Landed`, `nil`                             |
+| `AVAILABILITY`   | - Availability of the fosterer<br/> - Case-sensitive<br/> - Can be Available / NotAvailable / nil                            | `NotAvailable`, `Available`, `nil`                          |
```

Comment:
do u think we can remove the part about "can be HDB..." and for availability too! i feel like it might be a repeat :D

Reply:
same for the one in edit too!

Reply:
wait its not mentioned anywhere else tho...?